usaspending_api/disaster/v2/views/object_class/spending.py

```diff
@@ -44,13 +44,13 @@ class ObjectClassSpendingViewSet(PaginationMixin, SpendingMixin, FabaOutlayMixin
     def post(self, request):
         if self.spending_type == "award":
             results = list(self.award_queryset)
-            include_award_count = True
+            extra_columns = ["award_count"]
         else:
             results = list(self.total_queryset)
-            include_award_count = False
+            extra_columns = []
```

Comment:
Just following the pattern for `federal_account/spending.py` and `agency/spending.py` ... shouldn't this be 
```
extra_columns = ["total_budgetary_resources"]
```
Like those?

Reply:
nvm. Object Class doesn't show budget since GTAS (where budget comes from) doesn't have object class dimensionality.